server/controllers/cohort.js

```diff
@@ -441,16 +514,15 @@ const addMember = (req, resp) => {
         );
       }
 
-      const { _id, avatarUrl, displayName } = user;
+      const { _id } = user;
 
       if (checkIfCohortMember(currentCohort, _id)) {
         throw new BadRequestException(
           'cohort.actions.add-member-already-member'
         );
       }
-
       currentCohort.memberIds.push(_id);
-      newMember = { avatarUrl, _id, displayName };
+      newMember = user;
 
       return currentCohort.save();
```

Comment:
you can also use helper `returnPayload` here to return the actual user data

Reply:
In this case, I can't use `returnPayload`, because I have to rebuild this helper and refactor all its usage. In this case, I need cohort data and data return by the promise I passed to this helper.